libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -381,6 +381,10 @@ - (void)continueNativeBrowserFlow {
         }
 
     }];
+ 
+    if (@available(iOS 13.0, *)) {
+        _authSession.prefersEphemeralWebBrowserSession = true;
```

Comment:
- It looks like there's a compilation issue, should this be set on `_asWebAuthenticationSession` instead? 
- `true` -> `YES`?

Reply:
agh, sorry, that's what i get for cherry-picking

Reply:
@asikora-sf Could you also add a `TODO: Remove in Mobile SDK 9.0` comment here? We will go to base SDK `iOS 13` in `SDK 9.0` and we would want to remove the conditional then.